src/main/java/games/strategy/engine/framework/headlessGameServer/HeadlessGameServer.java

```diff
@@ -648,12 +651,127 @@ public Chat getChat() {
   }
 
   public static void main(final String[] args) {
-    GameRunner.handleCommandLineArgs(args, getProperties(), GameRunner.GameMode.HEADLESS_BOT);
+    System.getProperties().setProperty(GameRunner.TRIPLEA_HEADLESS, "true");
+    if (!ArgParser.handleCommandLineArgs(args, getProperties())) {
+      usage();
+      return;
+    }
+
+    handleHeadlessGameServerArgs();
     ClipPlayer.setBeSilentInPreferencesWithoutAffectingCurrent(true);
     try {
       new HeadlessGameServer();
     } catch (final Exception e) {
       ClientLogger.logError("Failed to start game server: " + e);
     }
   }
+
+  private static void usage() {
+    System.out.println("\nUsage and Valid Arguments:\n"
+        + "   " + GameRunner.TRIPLEA_GAME_PROPERTY + "=<FILE_NAME>\n"
+        + "   " + GameRunner.TRIPLEA_GAME_HOST_CONSOLE_PROPERTY + "=<true/false>\n"
+        + "   " + GameRunner.TRIPLEA_SERVER_PROPERTY + "=true\n"
+        + "   " + GameRunner.TRIPLEA_PORT_PROPERTY + "=<PORT>\n"
+        + "   " + GameRunner.TRIPLEA_NAME_PROPERTY + "=<PLAYER_NAME>\n"
+        + "   " + GameRunner.LOBBY_HOST + "=<LOBBY_HOST>\n"
+        + "   " + LobbyServer.TRIPLEA_LOBBY_PORT_PROPERTY + "=<LOBBY_PORT>\n"
+        + "   " + GameRunner.LOBBY_GAME_COMMENTS + "=<LOBBY_GAME_COMMENTS>\n"
+        + "   " + GameRunner.LOBBY_GAME_HOSTED_BY + "=<LOBBY_GAME_HOSTED_BY>\n"
+        + "   " + GameRunner.LOBBY_GAME_SUPPORT_EMAIL + "=<youremail@emailprovider.com>\n"
+        + "   " + GameRunner.LOBBY_GAME_SUPPORT_PASSWORD + "=<password for remote actions, such as remote stop game>\n"
+        + "   " + GameRunner.LOBBY_GAME_RECONNECTION + "=<seconds between refreshing lobby connection [min "
+        + GameRunner.LOBBY_RECONNECTION_REFRESH_SECONDS_MINIMUM + "]>\n"
+        + "   " + GameRunner.TRIPLEA_SERVER_START_GAME_SYNC_WAIT_TIME
+        + "=<seconds to wait for all clients to start the game>\n"
+        + "   " + GameRunner.TRIPLEA_SERVER_OBSERVER_JOIN_WAIT_TIME
+        + "=<seconds to wait for an observer joining the game>\n"
+        + "   " + GameRunner.MAP_FOLDER + "=mapFolder"
+        + "\n"
+        + "   You must start the Name and HostedBy with \"Bot\".\n"
+        + "   Game Comments must have this string in it: \"automated_host\".\n"
+        + "   You must include a support email for your host, so that you can be alerted by lobby admins when your "
+        + "host has an error."
+        + " (For example they may email you when your host is down and needs to be restarted.)\n"
+        + "   Support password is a remote access password that will allow lobby admins to remotely take the "
+        + "following actions: ban player, stop game, shutdown server."
+        + " (Please email this password to one of the lobby moderators, or private message an admin on the "
+        + "TripleaWarClub.org website forum.)\n");
+  }
+
+  private static void handleHeadlessGameServerArgs() {
+    boolean printUsage = false;
```

Comment:
This was the bot specific code previously in game runner handle command line args.